UnityProject/Assets/Scripts/UI/ChangeLogs/ChangeLogHandler.cs

```diff
@@ -47,8 +47,9 @@ void OpenWindow()
 
     public void CloseWindow()
     {
-        SoundManager.Play("Click01");
-        window.SetActive(false);
+		// JESTER
+		//SoundManager.Play("Click01");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)